ckoga/SWAPI-Study-Sesh
# Header

### What's this PR do?  
- Finalizes where/how the header component will be rendered (in App conditionally based on this.state.user).
- Add method to update the state with the user info as an object.
- Styling of the Header can be updated as we finalize CSS updates

### Where should the reviewer start?
- Check out the `handleClick()` method in WelcomeForm and see how it fires the `updateUser()` method within App which renders the Header component conditionally.

### How should this be manually tested?
Pull down to your machine:
- try clicking Enter button without entering a name
- enter a name then click Enter, see how the Header renders with the user info
- use the React dev tools to see how the state of Welcome Form and App update during the previous steps

### What are the relevant tickets?
This closes issue #5
